Library/Homebrew/rubocops/all.rb

```diff
@@ -1,17 +1,9 @@
 # typed: strict
 # frozen_string_literal: true
 
+# TODO: remove this (and avoid further active support in rubocops)
 require "active_support/core_ext/array/conversions"
 
-require "rubocop-performance"
-require "rubocop-rails"
-require "rubocop-rspec"
-
-require_relative "../warnings"
-Warnings.ignore :parser_syntax do
```

Comment:
we should no longer need this wrapper, bc it will happen in `rubocops/shared/helper_functions.rb` before we get to requiring `rubocop-sorbet` in `.rubocop.yml`.